lib/ansible/plugins/action/include_vars.py

```diff
@@ -80,6 +86,10 @@ def _set_args(self):
     def run(self, tmp=None, task_vars=None):
         """ Load yml files recursively from a directory.
         """
+        if tmp is not None:
+            display.warning('tmp is no longer a supported parameter of ActionModule.run().  It has no effect')
```

Comment:
you dont need to add display, just return warning in result

Reply:
I can do that but it adds a lot of boilerplate to those modules.   (Because you need to merge the warnings with whatever's already in results).  Some of these modules already use display.XXX as well.  Perhaps I should just use results for the few modules which don't have display already imported?